ext/reflection/php_reflection.stub.php

```diff
@@ -48,6 +48,9 @@ public function getClosureThis() {}
     /** @return ReflectionClass|null */
     public function getClosureScopeClass() {}
 
+    /** @return ReflectionClass|null */
+    public function getClosureCalledClass() {}
```

Comment:
The return type could have been declared directly, since it's a brand new function (so there is no BC impact). :) Then there would be no need for the `@tentative-return-type` annotation in PHP-8.1.